vagrant/provisioning/roles/arkcase-app/templates/server.yml

```diff
@@ -159,3 +162,8 @@ payment:
     securepaylinkendpoint: "{{ touchnet_securepaylinkendpoint | default('') }}"
     upaysiteid: "{{ touchnet_upaysiteid | default('') }}"
 microsoftOfficeOnline.isWopiEnabled: {{ arkcase_ee | default('false') }}
+
+#### Configuration variables for PDFTron integration ###########
+pdftron:
+  viewer:
+    license: "{{ pdftron_viewer_license | default('') }}"
```

Comment:
Why do we need `pdftron_viewer_license` in two places?

Reply:
Don't really know, it was requirements from Nadica, those lines were in .arkcase MR's that she pushed.